tutorial.ipynb

```diff
@@ -80,13 +86,21 @@
    "outputs": [],
    "source": [
     "device = \"cpu\"\n",
+    "\n",
+    "if Version(\"0.26\") <= Version(gymnasium.__version__) < Version(\"1.0\"):\n",
```

Comment:
Is the lowerbound check here necessary? If lower versions are not supported, should just throw an error.

Reply:
Good catch, lower versions should not exist.

I will keep for now as-is, as this is consistent with the way we do it in the HumEnv. If we want to change that, let's do a dedicated change in both repos.